api/kyverno/v1/image_verification_types.go

```diff
@@ -183,7 +183,7 @@ type StaticKeyAttestor struct {
 	Secret *SecretReference `json:"secret,omitempty" yaml:"secret,omitempty"`
 
 	// Rekor provides configuration for the Rekor transparency log service. If the value is nil,
-	// Rekor is not checked. If an empty object is provided the public instance of
+	// or an empty object is provided, the public instance of
```

Comment:
@vishal-chdhry - so can a user specify that Rekor should not be used?

Reply:
Can they specify `ignoreTlog` to achieve this?

Reply:
Yes, If a user set `ignoreTlog` to be true, rekor will not be used